omegaconf/nodes.py

```diff
@@ -143,8 +143,8 @@ def __init__(
         super().__init__(
             parent=parent,
             value=value,
-            metadata=Metadata(
-                ref_type=None, object_type=None, key=key, optional=is_optional
+            metadata=Metadata(  # ref_type should be Optional[Type[Any]]
```

Comment:
What is this comment telling the reader?

Reply:
oh, I should have checked that comment, I think that i was having an error when type checking. I'll look into it.